RecoJets/JetPlusTracks/plugins/JetPlusTrackCorrector.cc

```diff
@@ -1385,8 +1385,6 @@ if( trBgOutOfVertex.size() == 0 ) return mScale;
 
     EnergyOfBackgroundCharged       = M_PI*mConeSize*mConeSize*EnergyOfBackgroundCharged;
     ResponseOfBackgroundCharged     = M_PI*mConeSize*mConeSize*ResponseOfBackgroundCharged;
-    NumberOfBackgroundChargedVertex = M_PI*mConeSize*mConeSize*NumberOfBackgroundChargedVertex;
-    NumberOfBackgroundChargedCalo   = M_PI*mConeSize*mConeSize*NumberOfBackgroundChargedCalo;
```

Comment:
It looks like these two variables can be deleted entirely from this file. Also, please delete the commented-out lines where they are used.